illustrations/assets/index.md

```diff
@@ -7,143 +7,142 @@ layout: app
 
 <hr />
 
-<div class="u-clearfix">
+<div class="container">
```

Comment:
This adds a max width here where there previously wasn't one. Please double check that this doesn't look wonky on larger screen sizes, or else use `container-fluid` instead.

Reply:
Even changing it to container-fluid there seems to be a max width issue due to added padding. If I use p-x-0 it keeps everything aligned and doesn't look wonky